velero-plugins/common/types.go

```diff
@@ -76,6 +76,16 @@ const (
 	PVOriginalReclaimPolicy   string = "migration.openshift.io/orig-reclaim-policy" // Original PersistentVolumeReclaimPolicy
 )
 
+// DC-related labels/annotations
+const (
+	DCPodDeploymentLabel       string = "deployment"                        // identifies associated RC
+	DCPodDeploymentConfigLabel string = "deploymentconfig"                  // identifies associated DC
+	DCPodDisconnectedLabel     string = "openshift.io/disconnected-from-dc" // pod had DC labels removed on restore
```

Comment:
I think pod labels should be oadp specific. Don't want to have cross-project label conflicts.

Reply:
or openshift-velero-plugin specific. velero.openshift.io?

Reply:
Is there a more specific label namespace we should be using for OADP labels? Looking at the current code, it seemed like we were using `openshift.io` and not something like `oadp.openshift.io` -- @dymurray what is the standard for OADP namespace for labels and annotations?

Reply:
I know for migration use cases we started with `openshift.io` and eventually moved to `migration.openshift.io` -- Whatever we use, we should standardize on the appropriate label/annotation namespace for OADP in general. Currently we seem to be using only "openshift.io" and "migration.openshift.io" -- and nothing else.

Reply:
@sseago @kaovilai we use the label `openshift.io/oadp` for OADP Operator resources https://github.com/openshift/oadp-operator/blob/master/api/v1alpha1/oadp_types.go#L31

Reply:
@shubham-pampattiwar So we're not using an oadp-specific namespace there it seems -- i.e. there is no oadp.openshift.io in use -- does this mean that "openshift.io/disconnected-from-dc" is correct, or should we introduce an oadp domain here? i.e. "oadp.openshift.io/disconnected-from-dc" -- @dymurray  any opinions here?

Reply:
Can we nest under /oadp?
`openshift.io/oadp/disconnected-from-dc`

Reply:
ok alternatively `openshift.io/oadp_disconnected-from-dc`

Reply:
I don't like it nested, i like the namespacing. `oadp.openshift.io/disconnected-from-dc` looks good to me. 

In general I like moving all our annotations/labels to `oadp.openshift.io` as that's the namespace our APIs live under.

Reply:
@dymurray or should it be adp.openshift.io? I'm a bit unclear on when to use "oadp" vs "adp" -- i.e. namespace is "openshift-adp" presumably because we don't want "openshift" mentioned twice in the name (once spelled out, once in the acronym). Would the same apply to the ".openshift.io" namespacing?

Reply:
Whatever it is, I'm assuming we'd use this for all new oadp label/annotation names going forward. "oadp.openshift.io" is probably clearest.